core/modules/qproc/SConscript

```diff
@@ -6,38 +6,42 @@ import os
 
 standardModule(env, test_libs='log4cxx')
 
+
 # build kvmap.h
 # If you need to rebuild the map, use:
 # ./admin/bin/qserv-admin.py < admin/examples/testMap_generateMap
 def charArrayGenerator(filename):
     bSize = 70
-    with open(filename,"rb") as f:
+    with open(filename, "r") as f:
```

Comment:
I pondered this. The problem is that python 3 `ord()` takes characters and not bytes, implying that you are actually wanting to convert characters to their hex equivalents. I think this is because for bytes, the iterator returns the integer as it goes. So if you keep byte mode you need to change the `hex(ord(x))` to straight `hex(x)`.

I can change it to do that if you prefer but I don't think that works on python2. The current code works on both (Qserv can drop python3 as soon as you get the admin scripts converted).